### DISCUSSION TITLE
Can I generate a x509 certificate using dilithium in C language using Openssl ?

### DISCUSSION THREAD

**Discussion by Aswin2203** [open-quantum-safe/oqs-provider#555] [General]
Hi, I have a Question would I be able to generate a x509 certficate using dilithium now called ML_DSA key pairs in C language using openssl or is there any approaches that could help me accomplish my tasks. 
OR the only possible way to generate is through CLI

**Comment by baentsch** [comment]
This is easily possible using OpenSSL's EVP API; (hence) this is not an oqs(provider) question.

**Comment by Aswin2203** [comment]
can you explain it more breifly
